docs/overview.md

```diff
@@ -67,14 +70,13 @@ architecture as [Index-While-Building][] in XCode 9.
   * [ ] dependency -> dependency.
 * [x] Find references (`textDocument/references`). Works the same as goto
       definition.
-  * [x] project -> project.
-  * [ ] project -> Scala dependency.
-  * [ ] project -> Java dependency.
-  * [ ] dependency -> dependency.
+  * [x] project references
+  * [ ] Scala dependency references
+  * [ ] Java dependency references
 * [x] Highlight references to symbol at position
       (`textDocument/documentHighlight`).
-* [x] Open symbol in file (`textDocument/documentSymbol`)
-* [x] Open symbol in workspace (`workspace/symbol`)
+* [x] Goto symbol in file (`textDocument/documentSymbol`)
```

Comment:
I think that here and below with the "outline in the sidebar" you kind of rely too much on a particular UI representation of this feature. It may vary from one client to other while LSP defines as

> a flat list of all symbols found in a given text document

(which is a bad (but simple) definition if you ask me, but still)

Anyway, don't mind it, I just wanted to note this nuance.